src/App.js

```diff
@@ -1,50 +1,36 @@
-import React, { useEffect } from "react";
-import _nav from "./_nav";
-import Fragment from "./Fragment";
-import Payback from "./Payback";
-import Search from "./Search";
-import 'bootstrap/dist/css/bootstrap.min.css'
-import { useState } from "react";
+import { useEffect, useState } from 'react'
 import axios from 'axios'
-import 'bootstrap'
+import Nav from './Nav'
+import Fragment from './Fragment'
+import Payback from './Payback'
+import Search from './Search'
 
-function App() {
-    const [price, setPrice] = useState(0)
-    const [page, setPage] = useState("fragment")
-
-    const displayPage = {"fragment": showFragment, "payback": showPayback, "search": showSearch}
-    
-    function showFragment() {
-        return(<Fragment />)
-    }
-
-    function showPayback() {
-        return(<Payback />)
-    }
-
-    function showSearch() {
-        return(<Search />)
-    }
+const pageUrl =
```

Comment:
This can be taken outside the component as a const that will never change. Therefore, we cant skip it from the `useEffect` dependency check